ext/phar/tests/bug69958.phpt

```diff
@@ -10,8 +10,8 @@ $tarphar = new PharData(__DIR__.'/bug69958.tar');
 $phar = $tarphar->convertToData(Phar::TAR);
 ?>
 --EXPECTF--
-Fatal error: Uncaught BadMethodCallException: phar "%s/bug69958.tar" exists and must be unlinked prior to conversion in %s/bug69958.php:%d
+Fatal error: Uncaught BadMethodCallException: phar "%sbug69958.tar" exists and must be unlinked prior to conversion in %sbug69958.php:%d
```

Comment:
Why is this is fix, when test case is marked as XFAIL anyway?

Reply:
probably to reduce the diff, it is evident the Windows paths has `\` instead of `/`

cherrypicked from https://github.com/php/php-src/pull/8392 and coded some time ago...